src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py

```diff
@@ -654,6 +654,223 @@ def test_provider_operation(self):
         ])
 
 
+class TemplateSpecsTest(LiveScenarioTest):
```

Comment:
May I know the reason using LiveScenarioTest? It is hard to record and play back?

Reply:
It is no longer a LiveScenarioTest

Reply:
Export template specs needs to be a live scenario test, it has been separated from the others.